<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <class>QtRegisterView</class>
 <widget class="QWidget" name="QtRegisterView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>用户注册</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>注册新账号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="newUsernameLabel">
     <property name="text">
      <string>用户名:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="newUsernameEdit">
     <property name="placeholderText">
      <string>请输入新用户名</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="newPasswordLabel">
     <property name="text">
      <string>密码:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="newPasswordEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>请输入密码</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="confirmPasswordLabel">
     <property name="text">
      <string>确认密码:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="confirmPasswordEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>请再次输入密码</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="registerSubmitButton">
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="backToLoginButton">
     <property name="text">
      <string>返回登录</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>